kustomize/install/default/kustomization.yaml

```diff
@@ -12,14 +12,9 @@ bases:
 
 images:
 - name: postgres-operator
-  newName: registry.developers.crunchydata.com/crunchydata/postgres-operator
-  newTag: ubi8-5.3.0-0
-- name: postgres-operator-upgrade
-  newName: registry.developers.crunchydata.com/crunchydata/postgres-operator-upgrade
-  newTag: ubi8-5.3.0-0
+  newName: us.gcr.io/crunchy-container-testing/tl-postgres-operator
+  newTag: ubi8-5.3.0
```

Comment:
This is an interesting case: we're adding the upgrade manager to the pgo binary, but we don't have that image available to the public, so if they run 5.3.0, it'll run PGO but not do upgrades. So our choice is: 
a) put the old 5.3.0 image in, knowing it'll only do half of what we want it to do; or 
b) put in a not-available image / nonsense, knowing it won't be available

Since this is `devel` branch, which is meant to be up-to-date with `postgres-operator`, I'm inclined to leave this as it currently is -- option (b) -- and maybe update the image in the values.yaml to match.

My thought process here is: someone should not use the `devel` branch without forethought; and a crashed deployment (because of image pull error) will more effectively stop someone from using this rather than allow them to go forward and then ask "why is upgrade not working?"

What do you think?

Reply:
I like option (b), too.

Reply:
I'm on board with option (b), but I think I'd like it to be something like

```yaml
  newName: registry.developers.crunchydata.com/crunchydata/postgres-operator
  newTag: unreleased
```